module/VuFind/src/VuFind/ILS/Driver/Alma.php

```diff
@@ -706,7 +706,7 @@ public function createAlmaUser($formParams)
      * @param string $username The patrons barcode or other username.
      * @param string $password The patrons password.
      *
-     * @return string[]|NULL
+     * @return string[]|null
```

Comment:
It's not really accurate to call this a `string[]` array since it may contain mixed types. Probably best to use the generic array instead for now:

```suggestion
     * @return ?array
```

Note that most of the other ILS drivers actually just use a `mixed` return type on this method. I think it might be beneficial to change all of them to `?array` for greater specificity, though that's a project beyond the scope of this current PR.

Reply:
done